Library/Formula/wdt.rb

```diff
@@ -0,0 +1,74 @@
+class Wdt < Formula
+  desc "Transfer data as fast as possible over multiple TCP paths."
+  homepage "https://github.com/facebook/wdt"
+
+  stable do
+    url "https://github.com/facebook/wdt/archive/v1.26.1603040.tar.gz"
+    sha256 "b0fffdd12478ef74de50852b9c356352dae5f7ccee22dbca91a88afc6c5b6fee"
+
+    resource "folly" do
+      url "https://github.com/facebook/folly/archive/v0.57.0.tar.gz"
+      sha256 "92fc421e5ea4283e3c515d6062cb1b7ef21965621544f4f85a2251455e034e4b"
+    end
+
+    patch :DATA
+  end
+
+  head do
+    url "https://github.com/facebook/wdt.git"
+
+    resource "folly" do
+      url "https://github.com/facebook/folly.git"
+    end
+  end
+
+  depends_on "cmake" => :build
+  depends_on "gflags" => :build
+  depends_on "glog" => :build
+  depends_on "double-conversion" => :build
+  depends_on "openssl"
+  depends_on "folly"
+  depends_on "boost"
+
+  needs :cxx11
+
+  def install
+    ENV.cxx11
+    ENV.libcxx if ENV.compiler == :clang
+
+    args = %w[
+      -DOPENSSL_ROOT_DIR=#{Formula["openssl"].opt_prefix}
+    ]
+
+    (buildpath/"folly").install resource("folly")
+
+    args << "-DFOLLY_SOURCE_DIR=" + buildpath/"folly"
+
+    mktemp do
+      mkdir "wdt"
+      cp_r Dir["#{buildpath}/."], "wdt"
+
+      system "cmake", buildpath, *(std_cmake_args + args)
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "#{bin}/wdt", "--version"
```

Comment:
Sure, I have tried to write a test, but it always failed in the test sandbox, even I can use same command in the homebrew debug shell. So, I have to remove it, because most of guys just need wdt tools instead of it's library.

Could you help me to figure out the root cause?

```ruby
  test do
    system "#{bin}/wdt", "--version"

    (testpath/"test.cpp").write <<-EOS.undent
      #include <wdt/Wdt.h>
      using namespace facebook::wdt;
      int main() {
        Wdt::initializeWdt("test");
        return 0;
      }
    EOS

    ENV.cxx11

    system ENV.cxx, "test.cpp", "-L#{lib}", "-o", "test",
                    "-I#{Formula["openssl"].opt_include}",
                    "-L#{HOMEBREW_PREFIX}/lib",
                    "-lboost_system", "-lwdt"
    system "./test"
  end
```
it will compile test.cpp with command line, 
```
╰─ brew test wdt --debug
/usr/local/Library/brew.rb (Formulary::TapLoader): loading /usr/local/Library/Formula/wdt.rb
Testing wdt
...
/usr/local/Library/Homebrew/debrew.rb:11:in `raise'
BuildError: Failed executing: /usr/bin/clang++ -std=c++11 -stdlib=libc++ test.cpp -L/usr/local/Cellar/wdt/1.26.1603040/lib -o test -I/usr/local/opt/openssl/include -L/usr/local/lib -lboost_system -lwdt
1. raise
2. ignore
3. backtrace
4. irb
5. shell
Choose an action: 5
When you exit this shell, you will return to the menu.
localhost% /usr/bin/clang++ -std=c++11 -stdlib=libc++ test.cpp -L/usr/local/Cellar/wdt/1.26.1603040/lib -o test -I/usr/local/opt/openssl/include -L/usr/local/lib -lboost_system -lwdt
localhost% ./test
WARNING: Logging before InitGoogleLogging() is written to STDERR
I0325 10:10:43.505951 2095534080 EncryptionUtils.cpp:82] Openssl library initialized
I0325 10:10:43.506959 2095534080 Wdt.cpp:22] One time initialization of WDT for test
I0325 10:10:43.506976 2095534080 WdtFlags.cpp:63] Running WDT 1.26.1603040 p 26
```